gpflow/quadrature/gauss_hermite.py

```diff
@@ -78,12 +78,15 @@ def __init__(self, dim: int, n_gh: int):
         :param n_gh: number of Gauss-Hermite points, integer
         :param dim: dimension of the multivariate normal, integer
         """
+        print("PRINT - Initializing NDiagGHQuadrature")
+        tf.print("TF PRINT - Initializing NDiagGHQuadrature")
```

Comment:
Good for testing & developing, but shouldn't be part of the release
```suggestion
```

Reply:
Removed.